### DISCUSSION TITLE
Why should the image sizes be divisible by 8 ?

### DISCUSSION THREAD

**Discussion by tcourat** [zju3dv/LoFTR#272] [Q&A]
The model seems to run fine even when image sizes are not divisible by 8. Then why is it stated that it should be divisible by 8 ?

**Comment by williamhoole** [comment]
The reason for this is because of the Resolution in the coarse matching. This means that a coarse match is done at every 8 pixels. Another reason is due to the input into the Vision transformer as the token size is reduced. This is why it is recomended to use a Image size that is divisible by 8.
